dist/spec.json

```diff
@@ -15771,6 +15771,7 @@
           "$ref": "#/definitions/PolicyPeopleCondition"
         }
       },
+      "x-okta-parent": "#/definition/PolicyRuleConditions",
```

Comment:
probably, should be spelled definition with 's' at the end: definitions

Reply:
yes, thanks